framework/src/actions/MaterialOutputAction.C

```diff
@@ -225,8 +196,15 @@ MaterialOutputAction::act()
     params.set<MooseEnum>("family") = "MONOMIAL";
 
     // Create the AuxVariables
+    std::ostringstream oss;
     for (const auto & var_name : material_names)
+    {
+      oss << "\n  " << var_name;
       _problem->addAuxVariable("MooseVariableConstMonomial", var_name, params);
+    }
+    _console << COLOR_CYAN << "The following total " << material_names.size()
+             << " aux variables:" << oss.str() << "\nare added for automatic output by " << type()
+             << "." << COLOR_DEFAULT << std::endl;
```

Comment:
Why do you need this? You will see the aux variable listed in the traditional output location

Reply:
They could be mixed into other aux variables. Having this can make clearer what the output action exactly does. Does not hurt, right?

Reply:
OK, can you also add a test for this text.

Reply:
You did, thanks.